build/Build.Tests.cs

```diff
@@ -243,7 +243,7 @@ void RunIntegrationTests(string testFramework, string testRuntime)
         // By doing things this way, we can have a seamless experience between local and remote builds.
         var octopusTentacleTestsDirectory = BuildDirectory / "Octopus.Tentacle.Tests.Integration" / testFramework / testRuntime;
         var testAssembliesPath = octopusTentacleTestsDirectory.GlobFiles("*.Tests*.dll");
-        var testResultsPath = ArtifactsDirectory / "teamcity" / $"TestResults-{testFramework}-{testRuntime}.xml";
+        var testResultsPath = ArtifactsDirectory / "teamcity" / $"TestResults-Tests-Integration-{testFramework}-{testRuntime}.xml";
```

Comment:
Nit: would it be more consistent to name this `TestResults-IntegrationTests-{testFramework}-{testRuntime}.xml`